### PR TITLE
Survey project

### PR DESCRIPTION
**Pull request by leelalaim**
Link to my deployed project: https://friend-match.netlify.app/

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,165 @@
+import { useState } from 'react';
+
+import DropdownLocation from 'DropdownLocation'
+import NameInput from 'NameInput'
+import RadioBtnAge from 'RadioBtnAge'
 import React from 'react'
+import Summary from 'Summary'
+import Header from './Header'
+
+import './index.css';
```
Comment (matildasvensson85) [inline code/src/App.js:10]: Sometimes semicolons are used and sometimes not

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,165 @@
...
+              className='back-button'
+              type='button'
+              value='welcome'
+              onClick={event => { setSection(event.target.value) }}
+            >
+              Back
+            </button>
+            <button
+              className='next-button'
+              type="submit"
```
Comment (matildasvensson85) [inline code/src/App.js:76]: Sometimes " is used and sometimes '

File: code/src/DropdownLocation.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+const DropdownLocation = ( {label, location, onLocationChange}  ) => {
+    return (
+        <div className='dropdown-location'>
```
Comment (matildasvensson85) [inline code/src/DropdownLocation.js:5]: Can <section> be used here instead, to make the html more semantic?

**Review by matildasvensson85 — COMMENTED** [review]
Really nice website turning the survey into a friend matching website. The requirements fpr blue level is well fulfilled. 

The code is well structured and organized, and easy to understand. The indentation and formatting looks good. There are some inconsistencies when it comes to semicolons and single or double quotes. The site is accessible, a lot of semantic tags are used, and the code is made  even more accessible through aria code. 

The styling looks really good and worked through, and the site is responsive. The only thing Im thinking that could possibly be changed is that when you resize the window, there is a moment where the text  is half on the black part of the screen, and half on the image. This makes the readability of the text not optimized, since the font color is white, and the colors of the sky is quite light as well.

**Review by matildasvensson85 — CHANGES_REQUESTED** [review]
Really nice website turning the survey into a friend matching website. The requirements fpr blue level is well fulfilled. 

The code is well structured and organized, and easy to understand. The indentation and formatting looks good. There are some inconsistencies when it comes to semicolons and single or double quotes. The site is accessible, a lot of semantic tags are used, and the code is made  even more accessible through aria code. 

The styling looks really good and worked through, and the site is responsive. The only thing Im thinking that could possibly be changed is that when you resize the window, there is a moment where the text  is half on the black part of the screen, and half on the image. This makes the readability of the text not optimized, since the font color is white, and the colors of the sky is quite light as well.

File: code/src/App.js (hunk 1 above)
Comment (KarolCislowski) [inline code/src/App.js:10]: Clean your imports up :) 
1. internal imports from react
2. components from node_modules
3. your components
4. CSS and another imports

**Review by KarolCislowski — APPROVED** [review]
Great job! 
For the next step, it would be nice to look for info on how to validate users' inputs :)

also, you should revisit your styling:
image

**Pull request closed without merging**